CONTRIBUTING.md

```diff
@@ -57,6 +57,23 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Documentation changes
+
+The documentation is published in form of a book at:
+
+- [Latest released version](https://cluster-api.sigs.k8s.io)
+- [Tip of the main branch](https://tip.cluster-api.sigs.k8s.io/)
+
+The source for the book is [this folder](https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/book/src) containing markdown files
+and we use [mdBook](https://github.com/rust-lang/mdBook) to build it into an HTML
+version.
+
+After making changes locally you can run `make serve-book` which will build the HTML version
+and start a web server so you can preview if the changes render correctly at
+http://localhost:3000/.
```

Comment:
```suggestion
http://localhost:3000; the preview auto-updates when changes are detected.
```

Reply:
Extra credit: we could mention that this downloads mdBook into `~/.cargo/bin` if you don't already have it (I think that's how it works).

Reply:
Done & done 😉